src/pages/Wallet.js

```diff
@@ -1,9 +1,34 @@
 import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
 
-class Wallet extends React.Component {
+export class Wallet extends React.Component {
```

Comment:
Mesmo coisa que eu apontei no arquivo `Login.js`, não há necessidade desse `export`

```suggestion
class Wallet extends React.Component {
```